lib/companies/pending_changes.ex

```diff
@@ -75,6 +76,18 @@ defmodule Companies.PendingChanges do
     Map.put(current_values, :industry_id, industry_id)
   end
 
+  defp current_values(%{changes: %{"id" => id}, resource: "job"}) do
+    {%{id: company_id}, current_values} =
+      Job
+      |> Repo.get(id)
+      |> Repo.preload([:company])
+      |> drop_ecto_fields()
+      |> drop_nulls()
+      |> Map.pop(:company)
+
+    Map.put(current_values, :company_id, company_id)
```

Comment:
If you drop the `preload` you'll be left with `company_id`, won't you?

Reply:
Damn, that's true. I guess when one does a pr @ 2 in the morning, this happens :)